myapp/app/models/task.rb

```diff
@@ -56,4 +60,14 @@ class Task < ApplicationRecord
   def set_user
     self.user ||= Current.user if self.new_record?
   end
+
+  def tag_list
+    tags.map(&:name).join(', ')
+  end
+
+  def tag_list=(names)
+    self.tags = names.split(',').map do |name|
```

Comment:
Probably a good idea to 
1. strip whitespaces
2. limit user input to alphanumeric
3. take a separate input for each tag
4. inform users how multiple tags are created

Reply:
Yes that will be better! 1-3 I think these are better implemented in FE and unfortunately I have lost all FE skills after being a dedicated BE for a long time 🙈 

and for 4 it's just I didn't create good documentation here..